platform/mbed_retarget.cpp

```diff
@@ -884,6 +884,9 @@ extern "C" WEAK void __iar_file_Mtxinit(__iar_Rmtx *mutex) {}
 extern "C" WEAK void __iar_file_Mtxdst(__iar_Rmtx *mutex) {}
 extern "C" WEAK void __iar_file_Mtxlock(__iar_Rmtx *mutex) {}
 extern "C" WEAK void __iar_file_Mtxunlock(__iar_Rmtx *mutex) {}
+#if (__IAR_SYSTEMS_ICC__ >= 8)
+extern "C" WEAK void *__aeabi_read_tp (void) {}
```

Comment:
Is an empty implementation _really_ sufficient? Do the TLS tests still pass with this empty implementation?

Reply:
TLS (Thread local storage support) is not yet added to mbed-os

Reply:
OK